src/main/java/net/education/kyivstar/config/PasswordEncryptor.java

```diff
@@ -0,0 +1,22 @@
+package net.education.kyivstar.config;
+
+import org.jasypt.encryption.pbe.StandardPBEStringEncryptor;
+
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.SECRET_WORD;
+
+public class PasswordEncryptor {
```

Comment:
Using static methods in this scenario isn't inherently bad, but there are some potential drawbacks, especially if you foresee needing to customize or extend the functionality in the future.
The PasswordEncryptor class is instantiated with a **secretWord** parameter, which sets up the encryptor with the provided secret word. This ensures that the encryptor is configured consistently throughout the usage of the class.
This approach allows you to have different instances of PasswordEncryptor with potentially different secret words, providing more flexibility.
This approach offers better encapsulation and allows for more flexible usage and potential customization in the future.

Reply:
What about this?